src/test/java/tests/SearchElements.java

```diff
@@ -11,10 +11,8 @@
 
     WebDriver driver;
 
-
-
     @Test
-    public void searchMethodTest(){
+    public void searchMethodTest(){ // First part of home work
         WebElement classname = driver.findElement(By.className("shopping_cart_link"));
         WebElement tagname = driver.findElement(By.tagName("body"));
         WebElement linktext = driver.findElement(By.linkText("LinkedIn"));
```

Comment:
Let's remove unused new lines in the list of elements @outthedj

Reply:
I've meant these)
![Screenshot_1](https://user-images.githubusercontent.com/13552476/153809981-01f180cd-12af-45c5-8f33-0e8bc5587652.png)